tests/acceptance/query-default-api-filter-test.js

```diff
@@ -4,63 +4,58 @@
  *  See the LICENSE file associated with the project for terms.
  */
 import EmberObject from '@ember/object';
-import { test } from 'qunit';
-import moduleForAcceptance from 'bullet-ui/tests/helpers/module-for-acceptance';
+import { module, test } from 'qunit';
 import RESULTS from '../fixtures/results';
 import COLUMNS from '../fixtures/columns';
 import FILTERS from '../fixtures/filters';
 import { jsonWrap } from '../helpers/pretender';
+import setupForAcceptanceTest from '../helpers/setup-for-acceptance-test';
+import { visit } from '@ember/test-helpers';
+import $ from 'jquery';
 
 let url = 'http://foo.bar.com/api/filter';
 let hit = 0;
 
-moduleForAcceptance('Acceptance | query default api filter', {
-  suppressLogging: true,
+module('Acceptance | query default api filter', function(hooks) {
+  setupForAcceptanceTest(hooks, [RESULTS.MULTIPLE], COLUMNS.BASIC);
 
-  beforeEach() {
+  hooks.beforeEach(function() {
     // Inject into defaultValues in routes, our mock filter values
-    this.application.register('settings:mocked', EmberObject.create({ defaultFilter: url }), { instantiate: false });
-    this.application.inject('route', 'settings', 'settings:mocked');
+    this.owner.register('settings:mocked', EmberObject.create({ defaultFilter: url }), { instantiate: false });
```

Comment:
Can we write another helper ```setupForMockSettings(hooks, defaultFilter)``` that sets up these beforeEach methods? It seems to be reused a bit.